pricejar/base/templates/base/contact.html

```diff
@@ -0,0 +1,113 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+  <div>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+    <link rel="stylesheet" href="style(ls).css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
+    <script src="https://kit.fontawesome.com/6d7c400781.js" crossorigin="anonymous"></script> 
+  </div>
+     
+  
+  <div style="background-color: #fff8f8;">
+
+    <!-- text -->
+    <div class=" px-5 mb-2 mx-auto m-5 " style="height:270px; width: 100%;">
+      <br>
+      <br>
+      <br>
+
+      <h1 class=" multicolor d-flex justify-content-center" > Contact Us</h1>
+      <p class="align-text-bottom d-flex justify-content-center">We will love to hear from you! Ask us a question or drop a comment , fill the form below and we'll get back to you as soon as we can.</p>
+
+    <!-- CARD!! -->  
+    <!-- 'm-5' margin .. 'mx-auto' center align -->
+    <div class="card px-5 mb-2 mx-auto m-5 " style=" width: 800px; height: 820px;">
+      <div class="card-body">
+
+    <div>
+      <form method='POST'>
+        <!-- Name input -->
+        <div class="form-outline mb-4">
+          <label class="form-label" for="loginName">Name</label>
+          <input type="name" id="loginName" class="form-control"  placeholder="Your name..."/>
```

Comment:
give your input the attribute 'name' with relevant values so that the input can be linked and processed at the back end.